docs/components/connectors/out-of-the-box-connectors/box.md

```diff
@@ -48,6 +48,10 @@ The Box Connector supports two methods for locating items stored in your Box acc
 
 :::
 
+:::note
+Starting from version 8.7.0, the Box Connector supports storing the response as a document. See additional details and limitations in [document handling](/components/concepts/document-handling.md).
+:::
+
```

Comment:
"... the Box Connector supports uploading documents from (or downloading documents to) the Camunda document store" (similar to S3)